### PR TITLE
[TF-3278] Improvements on typings

### PR DESCRIPTION
**Pull request by rhaylander**
**Jira link:** https://growtenfold.atlassian.net/browse/TF-3278

### CODE REVIEW

File: NodeJS/src/AgentClient.ts [hunk 1]
```diff
@@ -60,7 +54,7 @@ class AgentClient {
         agent_password: string,
         agentStationGroupId: any,
         agentLoginId: any,
-        skillIds: any, skillsWithPriority:[SkillPriority]) {
+        skillIds: any, skillsWithPriority: SkillPriority[]) {
```
Comment (reisandbeans) [inline NodeJS/src/AgentClient.ts:84]: this should be on a new line, I think

File: NodeJS/src/SampleAgentClient.ts [hunk 2]
```diff
@@ -1,8 +1,9 @@
 import {Err, Ok, Result} from "ts.data.json/dist/result";
 import * as Constants from "./Constants";
 import { createSession } from "./session";
-import SkillPriority, { createAgentClient} from "./AgentClient";
+import { createAgentClient} from "./AgentClient";
```
Comment (reisandbeans) [inline NodeJS/src/SampleAgentClient.ts]: Single quotes

File: NodeJS/src/SampleSubscriptionClient.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import * as Constants from "./Constants";
 import { createSession } from "./session"
 import { createSubscriptionClient } from "./SubscriptionClient";
-import {Subscription} from "./Subscription";
+import {DataDeliveryFormat, DataSourceType, RetryPolicy, Subscription} from "./definitions";
```
Comment (reisandbeans) [inline NodeJS/src/SampleSubscriptionClient.ts]: Missing spaces. And single quotes

File: NodeJS/src/Utils.ts [hunk 4]
```diff
@@ -1,5 +1,5 @@
-import {Err, JsonDecoder} from "ts.data.json";
-import SkillPriority from "./AgentClient";
+import { Err, JsonDecoder } from "ts.data.json";
```
Comment (reisandbeans) [inline NodeJS/src/Utils.ts:1]: single quotes

**Review by reisandbeans — APPROVED** [review]

**Comment by rhaylander** [comment]
These changes have already been merged to spokencloud/avaya-cloud-api, PR: spokencloud/avaya-cloud-api#14

**Pull request closed without merging**